src/main/java/com/stripe/model/Plan.java

```diff
@@ -107,6 +109,14 @@ public void setName(String name) {
 		this.name = name;
```

Comment:
Can you move the accessors for `name`, `statementDescriptor` and `trialPeriodDays` at the bottom of the list, with a comment like e.g. `@deprecated Prefer using the `product` field (https://stripe.com/docs/upgrades#2018-02-05)`?